src/main/kotlin/no/nav/syfo/avvistsykmelding/AvvistSykmeldingService.kt

```diff
@@ -0,0 +1,91 @@
+package no.nav.syfo.avvistsykmelding
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import kotlinx.coroutines.delay
+import no.nav.syfo.ApplicationState
+import no.nav.syfo.metrics.AVVIST_SM_MOTTATT
+import no.nav.syfo.metrics.AVVIST_SM_VARSEL_OPPRETTET
+import no.nav.syfo.model.ReceivedSykmelding
+import no.nav.syfo.objectMapper
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.time.LocalDate.now
+import java.time.LocalDateTime
+import java.util.Collections.singletonMap
+
+private val log: org.slf4j.Logger = LoggerFactory.getLogger("no.nav.syfo.syfosmvarsel")
+
+const val OPPGAVETYPE = "0005"
```

Comment:
Kan du legge på en kommentar om hva dette er og hva den peker på?

Reply:
http://stash.devillo.no/projects/FA/repos/mininnboks-tekster/browse/src/main/tekster/mininnboks/oppgavetekster 🙀

Reply:
Den teksten er litt misvisende for avvist sykmelding, men men 😅

Reply:
heh, jupp